package.json

```diff
@@ -1,41 +1,13 @@
 {
-  "name": "react-dates",
-  "version": "4.1.0",
+  "name": "react-dates-custom",
```

Comment:
If you're going to publish a fork, please use scoped modules instead of polluting npm's global namespace. You can also npm install from git.